src/OrchardCore.Modules/OrchardCore.Autoroute/Models/AutoroutePartExtensions.cs

```diff
@@ -0,0 +1,29 @@
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using Microsoft.Extensions.Localization;
+
+namespace OrchardCore.Autoroute.Models
+{
+    internal static class AutoroutePartExtensions
+    {
+        internal static IEnumerable<ValidationResult> ValidatePathFieldValue(this AutoroutePart autoroute, IStringLocalizer S)
```

Comment:
We need to check that the po extractor picks this up.

We know it finds, `var S = serviceProvider.Get<...>` but we don't know if it finds `S` in a method parameter.

Maybe @agriffard will check it?

Maybe @agriffard would also have to update all the po files so we don't lose the translations either.